content/posts/January-2021.md

```diff
@@ -26,3 +26,18 @@ weight: 79
 - We have temporarily disabled the testing-farm support because the cluster with old runner has died and the new runner is not ready.
   The tests are now skipped and you get `Testing farm is temporarily disabled.` status until we have the new runner available.
   ([packit-service#929](https://github.com/packit/packit-service/pull/929))
+
+## Week 4 (January 25th - January 29th)
+
+- CLI and service now support upstream/downstream default branch other than `master`.
```

Comment:
Would it be more clear this way?

```suggestion
- The CLI and the service now detects the name of the default branch of a repository instead of assuming to be called  `master`.
```